ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -406,15 +394,17 @@ getImmutableLedger ::
   => ChainDB m blk -> STM m (ExtLedgerState blk EmptyMK)
 getImmutableLedger = fmap LedgerDB.ledgerDbAnchor . getLedgerDB
 
--- | Get the ledger for the given point.
+-- | Get the ledger and changelog for the given point.
 --
 -- When the given point is not among the last @k@ blocks of the current
 -- chain (i.e., older than @k@ or not on the current chain), 'Nothing' is
 -- returned.
 getPastLedger ::
```

Comment:
Do we use this function in other places besides the mempool? If not we might consider making this definition local to the mempool.

Reply:
But we need to access things internal to the chain db. I think it should be fine to leave it where it is.